docs/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 ### Changed
 
 * Updated `syn`, `quote`, and `proc-macro2` to `1.0`. ([#1952])
+* Use a premultiplied view_proj matrix in vertex shaders. ([#1964])
```

Comment:
Sorry, you just added your changelog entry to the 0.13.1 section. You'll have to rebase and add it to the new Unreleased section.

Reply:
Done! Sorry about that.

Reply:
Sorry, I broke it again :P emergency patch release happened last night

Reply:
No problem, done!